build.gradle.kts

```diff
@@ -35,4 +42,20 @@ subprojects {
   kotlin {
     explicitApi()
   }
+
+  java {
+    withJavadocJar()
+    withSourcesJar()
+  }
+
+  publishing {
+    publications {
+      create<MavenPublication>("web5") {
+        groupId = project.group.toString()
```

Comment:
I see group as web5 and name as rootProject.name = "web5" 

so is this 
groupId = web5
artifactId = web5
?